packages/core/src/render/api.ts

```diff
@@ -65,6 +65,11 @@ export abstract class RendererFactory2 {
  * @publicApi
  */
 export abstract class Renderer2 {
+  /**
+   * Use to retrieve the base dom renderer instance.
+   */
+  abstract getBaseRenderer?(): Renderer2;
```

Comment:
nit 
```suggestion
  abstract getDelegateRenderer?(): Renderer2;
```

Thoughts on the name ?

Reply:
I changed it, but I think the name `delegate` is really only used in the animations package.